### ISSUE TITLE
add lable

### ISSUE THREAD

**Issue by hdihd9162** [estebanuri/facemaskdetector#6]
hello i need to add a label and show to user but i don't understand add to Which file can you help me

**Comment by lcarnevale** [comment]
Hey @hdihd9162,

You should get back to the model.
If I am right, the application uses a built model originally posted [here ](https://www.pyimagesearch.com/2020/05/04/covid-19-face-mask-detector-with-opencv-keras-tensorflow-and-deep-learning/). The model trains over a binary classified dataset (mask, no-mask).
I do not see a way to add a new label modifying the mobile application.

Best,
Lorenzo.
